bounce/server/resource/club.py

```diff
@@ -175,3 +175,19 @@ class SearchClubsResponse(metaclass=ResourceMeta):
             'minimum': 0,
         }
     }
+
+
+class ClubImageRequest(metaclass=ResourceMeta):
+    """
+    Defines the schema for a GET, PUT, DELETE
+    /clubs/<club_name>/images/<image_name> request.
+    """
+    __params__ = {
+        'club_name': {
+            'type': 'string',
+        },
+        'image_name': {
+            'type': 'string',
+            'default': '0',
+        }
+    }
```

Comment:
I don't think you need this resource class, because the club name and image name are part of the URI, but they're not actually request parameters so our resources middleware doesn't actually check them. If they were request parameters, then the URL would look more like this:

```
/clubs/images?club_name=<myclub>&image_name=<myimage>
```